src/plugins/intel_gpu/src/graph/layout_optimizer.cpp

```diff
@@ -1061,7 +1063,7 @@ format layout_optimizer::get_expected_format(convolution_node const& node) {
     const float cond_denom = _total_conv > 0 ? 1.0f / static_cast<float>(_total_conv) : 1.0f;
 
     bool onednn_valid_post_ops = get_post_ops_count(node) <= 32;
-    bool use_onednn_impls = _optimization_attributes.use_onednn_impls && input_layout.data_type != data_types::f32;
+    bool use_onednn_impls = _optimization_attributes.use_onednn_impls && input_layout.data_type != data_types::f32 && !node.weights_zero_points_term();
 
     if (use_onednn_impls && onednn_valid_post_ops) {
```

Comment:
What about like below? We can rely on previous result instead of adding `weights_zero_points_term` to our condition.
```suggestion
    bool use_onednn_impls = _optimization_attributes.use_onednn_impls && input_layout.data_type != data_types::f32;

    if (use_onednn_impls && onednn_valid_post_ops && node.get_preferred_output_fmt() != format::any) {
```